NetworkingDemo/DemoRepository.swift

```diff
@@ -30,7 +30,7 @@ internal class DemoRepository: AbstractRepository {
     }
     
     public func noAnswerEntities(userID: Int) -> SignalProducer<Void, RepositoryError> {
-        let path = DemoRepository.ReadPrefixPath + "/\(userID)" + DemoRepository.ReadSuffixPath
+        let path = DemoRepository.ReadPrefixPath + "\(userID)" + DemoRepository.ReadSuffixPath
```

Comment:
@danielaRiesgo 

I added a GH issue for this: https://github.com/Wolox/wolmo-networking-ios/issues/19

So this PR just contains codestats related changes.